queries.sql

```diff
@@ -172,4 +172,84 @@ WHERE animals.escape_attempts = 0 AND owners.full_name ='Dean Winchester';
 SELECT full_name, COUNT(*) FROM owners
 JOIN animals
 ON owners.id = animals.owner_id
-GROUP BY full_name;
\ No newline at end of file
+GROUP BY full_name;
+
+-- Who was the last animal seen by William Tatcher?
+ SELECT animals.name, vistis.visit_date FROM visits 
+ LEFT JOIN animals ON animals.id = visits.animal_id 
+ LEFT JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'William Tatcher' 
+ ORDER BY visits.visit_date
+ DESC LIMIT 1;
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT vets.name AS vet_name, species.name FROM vets
+
+LEFT JOIN specializations
+ON specializations.vet_id = vets.id
+LEFT JOIN species
+ON species.id = specializations.species_id;
+
+--How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT animals) AS different_animals_seen
+FROM vets
+WHERE name = 'Stephanie Mendez';
```

Comment:
- [ ] Great work adding the query for the question **How many different animals did Stephanie Mendez see?** as suggested by the previous reviewer :clap: However the best way to implement this would be to join the `visits`, `animals` and `vets` tables as in this SQL code.
```sql
  SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```